pkgs/development/python-modules/keyboard/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "keyboard";
+  version = "0.13.5";
+
+  src = fetchFromGitHub {
+    owner = "boppreh"; 
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-U4GWhPp28azBE3Jn9xpLxudOKx0PjnYO77EM2HsJ9lM=";
+  };
+
+  doCheck = false;
```

Comment:
pythonImportsCheck is required when disabling tests and also add a comment why you disabled tests.